ITRStartTask/StoryDetail/Views/StoryViewController.swift

```diff
@@ -38,6 +38,7 @@ class StoryViewController: UIViewController, XibInitializable {
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
         presenter.show()
```

Comment:
Show should be the last call in function

Reply:
fixed